### DISCUSSION TITLE
Unable to get cloudfare workers to work with telegraf.

### DISCUSSION THREAD

**Discussion by JunK4i** [telegraf/telegraf#1908] [Q&A] [answered]
Tried the linked middleware but it doesn't work! the webpack output doesn't seem to be syntatically acceptable to the cloudfare worker platform

**Comment by MKRhere** [comment, marked as answer]
Since the module hasn't been updated, I'm not sure if CF Workers has changed something about their API etc. Work is underway to make Telegraf v5 natively compatible with edge providers, but we're not done yet. Also #1817.

**Reply by JunK4i** [reply]
Ah thanks! Thanks for the hard work!
